### ISSUE TITLE
Drag'n Drop backend

### ISSUE THREAD

**Issue by antoinedavid** [contao/core-bundle#666]
Hi,
With contao 4.3.1, the drag'n Drop in the backend does not seem to work.
Best,

**Comment by leofeyer** [comment]
What exactly do you mean and how can it be reproduced in the online demo?

**Comment by antoinedavid** [comment]
Hi,

From the demo site of contao i've exactly the same issue the drag'n drom of liste (mode 4) doesn't works

See my capture

![image](https://cloud.githubusercontent.com/assets/16242303/21673512/99175d44-d329-11e6-910f-7a85e257159c.png)

**Comment by leofeyer** [comment]
Works for me. Maybe it is a browser issue? Some extension blocking the JavaScript?

**Comment by antoinedavid** [comment]
Firefox, IE11 with no extension (under Contao 4.2.5 i've the same browsers) => NOK
Chrome, Edge, Safari => OK

Maybe a futur update of Firefox solve the issue, strange bug.

**Comment by Toflar** [comment]
This has to be reopened. Indeed, drag n drop does not work in latest Firefox. Can be reproduced by trying to drag n drop any content element in the demo.

**Comment by qzminski** [comment]
I can confirm that the issue is present on 4.3.5. The drag and drop does not work under Firefox and IE 11.

**Comment by leofeyer** [comment]
Ok. Has anybody debugged this yet?

**Comment by Toflar** [comment]
Nope, I was only able to reproduce for now.

**Comment by ausi** [comment]
Can be fixed by adding the following line to [mootao.js](https://github.com/contao/core-bundle/blob/bcad83ee45bfa1e1c2071f257dff4427864880f4/src/Resources/public/mootao.js#L267) after line 267:

```js
options.dragOptions.unDraggableTags = [];
```

The `unDraggableTags` option from [Drag.js:49](https://github.com/mootools/mootools-more/blob/687363b141c7d6abb89e6716462bbd99545f81e5/Source/Drag/Drag.js#L49) includes `'button'` by default which seems to be the issue here because the drag handle is a `<button>` element.

**Comment by leofeyer** [comment]
Thank you @ausi. :kissing_heart:

**Comment by leofeyer** [comment]
@ausi Why not `options.dragOptions.unDraggableTags = ['input', 'a', 'textarea', 'select', 'option'];`?

**Comment by leofeyer** [comment]
Unfortunately, the fix does not work. The problem seems to be that `this.idle` is always `false` in Firefox, which means that the `reset()` method is never executed.

**Comment by ausi** [comment]
I just tried it again on the *hotfix/4.3.8* branch, and it did fix the Firefox issue for me:

```diff
  initialize: function(lists, options) {
      options.dragOptions = Object.merge(options.dragOptions || {}, { preventDefault: (options.dragOptions && options.dragOptions.preventDefault) || Browser.Features.Touch });
+     options.dragOptions.unDraggableTags = [];
      return this.previous.apply(this, arguments);
  },
```

**Comment by leofeyer** [comment]
Just for the record, I have reported the issue to the MooTools devs in https://github.com/mootools/mootools-more/pull/1350.

**Comment by leofeyer** [comment]
Fixed in d61a54167658e3ff7acf86fefd77f1038e75dc9e. Thank you @ausi.
